models/classes/QtiCategoryPresetProvider.php

```diff
@@ -205,6 +188,69 @@ public function registerPresets(TestCategoryPresetProvider $presetService)
                 ])
             ]
         );
+
+        //TAO-7239, register new feature category only if it is enabled in config
+        /** @var \common_ext_ExtensionsManager $tt */
+        $extManager = $presetService->getServiceLocator()->get(\common_ext_ExtensionsManager::SERVICE_ID);
+        $testRunnerConfig = $extManager->getExtensionById('taoQtiTest')->getConfig('testRunner');
+
+        $isSkipaheadEnabled = isset($testRunnerConfig['test-taker-review-skipahead']) && $testRunnerConfig['test-taker-review-skipahead'];
+        if ($isSkipaheadEnabled) {
```

Comment:
Having a new option to disclose the feature, why not, but do you really need to duplicate the other options? Is it for the sort order?
It should be better to avoid such duplicate, especially since this introduces some mess and is error prone.
Why not something like:
```php
        $groupNavigation = [
            TestCategoryPreset::fromArray([
                'id'            => 'reviewScreen',
                'label'         => __('Enable Review Screen'),
                'qtiCategory'   => 'x-tao-option-reviewScreen',
                'description'   => __('Enable the item review screen / navigator.'),
                'order'         => 100,
                'pluginId'      => 'review'
            ]),
            TestCategoryPreset::fromArray([
                'id'            => 'markReview',
                'label'         => __('Enable Mark for Review'),
                'qtiCategory'   => 'x-tao-option-markReview',
                'description'   => __('Enable mark for review of items. Requires the Review Screen option.'),
                'order'         => 200,
                'pluginId'      => 'review'
            ]),
            TestCategoryPreset::fromArray([
                'id'            => 'informational',
                'label'         => __('Informational Item Usage'),
                'qtiCategory'   => 'x-tao-itemusage-informational',
                'description'   => __('Force the item to be considered as informational and not taken into account in (un)answered / flagged counters.'),
                'order'         => 300,
                'pluginId'      => 'review'
            ]),
            TestCategoryPreset::fromArray([
                'id'            => 'nextSection',
                'label'         => __('Allow Section Skipping'),
                'qtiCategory'   => 'x-tao-option-nextSection',
                'description'   => __('Allow skipping of the current section.'),
                'order'         => 400,
                'pluginId'      => 'nextSection',
                'featureFlag'   => 'next-section'
            ])
        ];
        if ($isSkipaheadEnabled) {
            $groupNavigation[] = TestCategoryPreset::fromArray([
                'id'            => 'skipAhead',
                'label'         => __('Enable Skipping Ahead'),
                'qtiCategory'   => 'x-tao-option-review-skipahead',
                'description'   => __('Enables skipping to items within this section. Requires the review screen option.'),
                'order'         => 250,
                'pluginId'      => 'review',
                'featureFlag'   => 'skip-ahead'
        }
        $presetService->register(
            TestCategoryPresetProvider::GROUP_NAVIGATION,
            $groupNavigation
        );
```

Reply:
Indeed, should do the job, will change this